Adopt more smart pointers in Network Process

RupinMittal opened the pull request:
#### 3d3c125c96828af610f140d759441215475328d6
<pre>
Adopt more smart pointers in Network Process
<a href="https://bugs.webkit.org/show_bug.cgi?id=285103">https://bugs.webkit.org/show_bug.cgi?id=285103</a>
<a href="https://rdar.apple.com/141945657">rdar://141945657</a>

Reviewed by Chris Dumez.

Smart pointer adoption as per the static analyzer.

* Source/WebCore/workers/service/server/SWServer.h:
(WebCore::SWServer::Connection::isWebSWServerConnection const):
* Source/WebCore/workers/service/server/SWServerToContextConnection.h:
* Source/WebKit/NetworkProcess/Cookies/WebCookieManager.cpp:
(WebKit::WebCookieManager::startObservingCookieChanges):
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp:
(WebKit::NetworkConnectionToWebProcess::initializeWebTransportSession):
* Source/WebKit/NetworkProcess/NetworkLoad.cpp:
* Source/WebKit/NetworkProcess/NetworkLoad.h:
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::didClose):
(WebKit::NetworkProcess::prepareToSuspend):
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::NetworkResourceLoader::startRequest):
(WebKit::NetworkResourceLoader::retrieveCacheEntry):
(WebKit::NetworkResourceLoader::processClearSiteDataHeader):
(WebKit::NetworkResourceLoader::continueWillSendRedirectedRequest):
* Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerFetchTask.cpp:
(WebKit::ServiceWorkerFetchTask::loadBodyFromPreloader):
(WebKit::ServiceWorkerFetchTask::convertToDownload):
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.cpp:
(WebKit::WebSWServerConnection::controlClient):
(WebKit::WebSWServerConnection::startFetch):
(WebKit::WebSWServerConnection::postMessageToServiceWorkerClient):
(WebKit::WebSWServerConnection::registerServiceWorkerClientInternal):
(WebKit::WebSWServerConnection::unregisterServiceWorkerClient):
(WebKit::WebSWServerConnection::computeThrottleState const):
(WebKit::WebSWServerConnection::updateThrottleState):
(WebKit::WebSWServerConnection::contextConnectionCreated):
(WebKit::WebSWServerConnection::sendToContextProcess):
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.h:
(isType):
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.h:
(isType):
* Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerToContextConnection.cpp:
(WebKit::WebSharedWorkerServerToContextConnection::WebSharedWorkerServerToContextConnection):
(WebKit::WebSharedWorkerServerToContextConnection::launchSharedWorker):
* Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.cpp:
(WebKit::NetworkRTCProvider::NetworkRTCProvider):
(WebKit::NetworkRTCProvider::attributedBundleIdentifierFromPageIdentifier):
* Source/WebKit/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebKit/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebKit/SaferCPPExpectations/UncountedLambdaCapturesCheckerExpectations:
* Source/WebKit/Shared/API/Cocoa/_WKRemoteObjectRegistry.mm:
(-[_WKRemoteObjectRegistry _callReplyWithID:blockInvocation:]):
* Source/WebKit/Shared/ApplePay/WebPaymentCoordinatorProxy.cpp:
(WebKit::WebPaymentCoordinatorProxy::showPaymentUI):
* Source/WebKit/UIProcess/AuxiliaryProcessProxy.cpp:
(WebKit::AuxiliaryProcessProxy::shutDownProcess):
* Source/WebKit/WebProcess/Storage/WebServiceWorkerFetchTaskClient.cpp:
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveRedirection):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveResponse):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveDataInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveFormDataAndFinishInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveBlobChunk):
(WebKit::WebServiceWorkerFetchTaskClient::didFailInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didFinishInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didNotHandleInternal):
(WebKit::WebServiceWorkerFetchTaskClient::usePreload):
(WebKit::WebServiceWorkerFetchTaskClient::contextIsStopping):
* Source/WebKit/WebProcess/Storage/WebServiceWorkerFetchTaskClient.h:

Canonical link: <a href="https://commits.webkit.org/288561@main">https://commits.webkit.org/288561@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e54ee609cf752cdc2989069b3405d8a3a8f4f3c2

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/83673 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/3290 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/37974 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/88744 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/34681 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/85758 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/3380 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/11248 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/65086 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/22915 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/86719 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/2489 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/76024 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/45374 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/2403 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/30234 "Found 1 new test failure: imported/w3c/web-platform-tests/css/css-grid/subgrid/subgrid-baseline-008.html (failure)") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/33730 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/73458 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/30979 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/90123 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/10938 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/7895 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/73522 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/11162 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/71849 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/72746 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/17003 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/15703 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/2262 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/12942 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/10890 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/16362 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/10738 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/14213 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/12510 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

RupinMittal commented on Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerToContextConnection.cpp:
```diff
@@ -142,8 +142,8 @@ void WebSharedWorkerServerToContextConnection::launchSharedWorker(WebSharedWorke
-    sharedWorker.forEachSharedWorkerObject([&](auto, auto& port) {
-        postConnectEvent(sharedWorker, port, [](bool) { });
+    sharedWorker.forEachSharedWorkerObject([connection = Ref { *this }, sharedWorker = Ref { sharedWorker }](auto, auto& port) {
+        connection->postConnectEvent(sharedWorker, port, [](bool) { });
```
Ref'ing `this` here is causing a crash with `ASSERTION FAILED: !m_adoptionIsRequired` (linked below). 

This is because in `WebSharedWorkerServerToContextConnection`'s constructor, we pass `*this` to `WebSharedWorkerServer::addContextConnection` and that un-adopted reference is eventually used by `WebSharedWorkerServer` to call this function here--resulting in this crash. 

How can I fix this? It seems wrong to use adoptRef in the constructor: `server.addContextConnection(adoptRef(*this));`

https://ews-build.s3-us-west-2.amazonaws.com/macOS-Sequoia-Debug-WK2-Tests-EWS/9d666430-6824/imported/w3c/web-platform-tests/content-security-policy/sandbox/shared-worker-sandbox-crash-log.txt

cdumez replied:
Call `relaxAdoptionRequirement();` in the class constructor.

cdumez commented on Source/WebKit/NetworkProcess/NetworkProcess.cpp:
```diff
@@ -237,14 +237,14 @@ void NetworkProcess::didClose(IPC::Connection&)
 
-    forEachNetworkSession([&](auto& session) {
-        platformFlushCookies(session.sessionID(), [callbackAggregator] { });
+    forEachNetworkSession([process = Ref { *this }, callbackAggregator = WTFMove(callbackAggregator)](auto& session) {
```
`process` -> `protectedThis` ?

cdumez commented on Source/WebKit/NetworkProcess/NetworkProcess.cpp:
```diff
@@ -2295,9 +2295,11 @@ void NetworkProcess::prepareToSuspend(bool isSuspensionImminent, MonotonicTime e
-        RELEASE_LOG(ProcessSuspension, "%p - NetworkProcess::prepareToSuspend() Process is ready to suspend", this);
-        UNUSED_VARIABLE(this);
+    RefPtr<CallbackAggregator> callbackAggregator = CallbackAggregator::create([weakThis = WeakPtr { *this }, completionHandler = WTFMove(completionHandler)]() mutable {
+        if (!weakThis)
```
Early return is definitely not OK here as we'd fail to call the completion handler

cdumez commented on Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerFetchTask.cpp:
```diff
@@ -583,8 +583,8 @@ bool ServiceWorkerFetchTask::convertToDownload(DownloadManager& manager, Downloa
     RefPtr<ServiceWorkerDownloadTask> serviceWorkerDownloadTask;
-    auto serviceWorkerDownloadLoad = NetworkLoad::create(*protectedLoader(), *session, [&](auto& client) {
-        serviceWorkerDownloadTask =  ServiceWorkerDownloadTask::create(*session, client, *serviceWorkerConnection, *m_serviceWorkerIdentifier, *m_serverConnectionIdentifier, m_fetchIdentifier, request, response, downloadID);
+    auto serviceWorkerDownloadLoad = NetworkLoad::create(*protectedLoader(), *session, [&serviceWorkerDownloadTask, &session, &serviceWorkerConnection, serviceWorkerIdentifier = m_serviceWorkerIdentifier, serverConnectionIdentifier = m_serverConnectionIdentifier, fetchIdentifier = m_fetchIdentifier, &request, &response, downloadID](auto& client) {
```
The new code is less readable. Why did we have to expand the `&` here?

RupinMittal replied:
The analyzer said that implicitly capturing `this` with the & was unsafe: https://build.webkit.org/results/Apple-Sequoia-Safer-CPP-Checks/288318@main%20(1002)/scan-build-output/StaticAnalyzer/WebKit/StaticAnalyzerReports/report-ServiceWorkerFetchTask.cpp-unknown-0-1b1184.html#EndPath

cdumez replied:
So don't use `this` inside the lambda and keep capturing `[&]` ?

RupinMittal replied:
Hmm, I'm confused about how I can avoid using `this`? For example, `serviceWorkerIdentifier` is a member of `this`, so unless it's captured explicitly by value in the lambda, `this` will be used.

But either way, I believe we use NOESCAPE here. Is this correct?

cdumez commented on Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.cpp:
```diff
@@ -235,10 +235,14 @@ void NetworkRTCProvider::stopResolver(LibWebRTCResolverIdentifier identifier)
 const String& NetworkRTCProvider::attributedBundleIdentifierFromPageIdentifier(WebPageProxyIdentifier pageIdentifier)
 {
-    return m_attributedBundleIdentifiers.ensure(pageIdentifier, [&]() -> String {
+    return m_attributedBundleIdentifiers.ensure(pageIdentifier, [checkedThis = CheckedPtr { this }, pageIdentifier]() -> String {
```
Seems we should create a `protectedThis` right away here instead of a `CheckedThis` since you end up synchronously constructing a protectedThis` a few lines below anyway.

cdumez commented on Source/WebKit/WebProcess/Storage/WebServiceWorkerFetchTaskClient.h:
```diff
@@ -86,9 +86,11 @@ class WebServiceWorkerFetchTaskClient final : public WebCore::ServiceWorkerFetch
+
+        Ref<WebServiceWorkerFetchTaskClient> protectedClient() { return client; }
 
         Ref<WebServiceWorkerFetchTaskClient> client;
```
can't we simply mark this as `const` to avoid the smart pointer warnings instead of having an unnecessary `protectedClient()`?

cdumez commented on Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.cpp:
```diff
@@ -446,8 +447,8 @@ void WebSWServerConnection::postMessageToServiceWorkerClient(ScriptExecutionCont
 
-    server->postMessageToServiceWorkerClient(destinationContextIdentifier, message, sourceIdentifier, sourceOrigin, [this] (auto destinationContextIdentifier, auto& message, auto sourceServiceWorkerData, auto& sourceOrigin) {
-        send(Messages::WebSWClientConnection::PostMessageToServiceWorkerClient { destinationContextIdentifier, message, sourceServiceWorkerData, sourceOrigin }, 0);
+    server->postMessageToServiceWorkerClient(destinationContextIdentifier, message, sourceIdentifier, sourceOrigin, [connection = Ref { *this }] (auto destinationContextIdentifier, auto& message, auto sourceServiceWorkerData, auto& sourceOrigin) {
```
`connection` -> `protectedThis`

cdumez commented on Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerToContextConnection.cpp:
```diff
@@ -142,8 +143,8 @@ void WebSharedWorkerServerToContextConnection::launchSharedWorker(WebSharedWorke
     send(Messages::WebSharedWorkerContextManagerConnection::LaunchSharedWorker { sharedWorker.origin(), sharedWorker.identifier(), sharedWorker.workerOptions(), sharedWorker.fetchResult(), initializationData });
-    sharedWorker.forEachSharedWorkerObject([&](auto, auto& port) {
-        postConnectEvent(sharedWorker, port, [](bool) { });
+    sharedWorker.forEachSharedWorkerObject([connection = Ref { *this }, sharedWorker = Ref { sharedWorker }](auto, auto& port) {
```
`connection` -> `protectedThis`

cdumez commented on Source/WebKit/NetworkProcess/Cookies/WebCookieManager.cpp:
```diff
@@ -156,8 +156,12 @@ void WebCookieManager::notifyCookiesDidChange(PAL::SessionID sessionID)
-        WebCore::startObservingCookieChanges(*storageSession, [this, sessionID] {
-            notifyCookiesDidChange(sessionID);
+        WebCore::startObservingCookieChanges(*storageSession, [weakThis = WeakPtr { *this }, sessionID] {
+            RefPtr protectedThis = weakThis.get();
```
Would be more concise without the early return:
```
if (RefPtr protectedThis = weakThis.get())
    protectedThis->notifyCookiesDidChange(sessionID);
```

cdumez commented on Source/WebKit/NetworkProcess/NetworkProcess.cpp:
```diff
@@ -237,14 +237,14 @@ void NetworkProcess::didClose(IPC::Connection&)
     ASSERT(RunLoop::isMain());
 
-    auto callbackAggregator = CallbackAggregator::create([this] {
+    auto callbackAggregator = CallbackAggregator::create([process = Ref { *this }] {
```
`process` -> `protectedThis`

cdumez commented on Source/WebKit/NetworkProcess/NetworkProcess.cpp:
```diff
@@ -2295,9 +2295,8 @@ void NetworkProcess::prepareToSuspend(bool isSuspensionImminent, MonotonicTime e
-    RefPtr<CallbackAggregator> callbackAggregator = CallbackAggregator::create([this, completionHandler = WTFMove(completionHandler)]() mutable {
-        RELEASE_LOG(ProcessSuspension, "%p - NetworkProcess::prepareToSuspend() Process is ready to suspend", this);
-        UNUSED_VARIABLE(this);
+    RefPtr<CallbackAggregator> callbackAggregator = CallbackAggregator::create([weakThis = WeakPtr { *this }, completionHandler = WTFMove(completionHandler)]() mutable {
```
Can omit `<CallbackAggregator>`

cdumez commented on Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
```diff
@@ -244,29 +245,30 @@ void NetworkResourceLoader::startRequest(const ResourceRequest& newRequest)
             WTF::switchOn(result,
-                [this] (ResourceError& error) {
-                    LOADER_RELEASE_LOG("start: NetworkLoadChecker::check returned an error (error.domain=%" PUBLIC_LOG_STRING ", error.code=%d, isCancellation=%d)", error.domain().utf8().data(), error.errorCode(), error.isCancellation());
+                [protectedThis] (ResourceError& error) {
```
Is there no way to mark the lambdas passed to `WTF::switchOn()` as `NOESCAPE` instead?

RupinMittal replied:
That's a good question. I'm not sure. @rniwa, do you know?

RupinMittal replied:
Seems like WTF::switchOn() should be hard-coded to ignore captured arguments.

rniwa commented on Source/WebKit/NetworkProcess/NetworkLoad.cpp:
```diff
@@ -61,7 +61,7 @@ NetworkLoad::NetworkLoad(NetworkLoadClient& client, NetworkLoadParameters&& para
 }
 
-NetworkLoad::NetworkLoad(NetworkLoadClient& client, NetworkSession& networkSession, const Function<RefPtr<NetworkDataTask>(NetworkDataTaskClient&)>& createTask)
+NetworkLoad::NetworkLoad(NetworkLoadClient& client, NetworkSession& networkSession, NOESCAPE const Function<RefPtr<NetworkDataTask>(NetworkDataTaskClient&)>& createTask)
```
You don't need to specify NOESCAPE in the function definition. Just put that on declaration.

RupinMittal committed:
Adopt more smart pointers in Network Process
https://bugs.webkit.org/show_bug.cgi?id=285103
rdar://141945657

Reviewed by Chris Dumez.

Smart pointer adoption as per the static analyzer.

* Source/WebCore/workers/service/server/SWServer.h:
(WebCore::SWServer::Connection::isWebSWServerConnection const):
* Source/WebCore/workers/service/server/SWServerToContextConnection.h:
* Source/WebKit/NetworkProcess/Cookies/WebCookieManager.cpp:
(WebKit::WebCookieManager::startObservingCookieChanges):
* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp:
(WebKit::NetworkConnectionToWebProcess::initializeWebTransportSession):
* Source/WebKit/NetworkProcess/NetworkLoad.cpp:
* Source/WebKit/NetworkProcess/NetworkLoad.h:
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::didClose):
(WebKit::NetworkProcess::prepareToSuspend):
* Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp:
(WebKit::NetworkResourceLoader::startRequest):
(WebKit::NetworkResourceLoader::retrieveCacheEntry):
(WebKit::NetworkResourceLoader::processClearSiteDataHeader):
(WebKit::NetworkResourceLoader::continueWillSendRedirectedRequest):
* Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerFetchTask.cpp:
(WebKit::ServiceWorkerFetchTask::loadBodyFromPreloader):
(WebKit::ServiceWorkerFetchTask::convertToDownload):
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.cpp:
(WebKit::WebSWServerConnection::controlClient):
(WebKit::WebSWServerConnection::startFetch):
(WebKit::WebSWServerConnection::postMessageToServiceWorkerClient):
(WebKit::WebSWServerConnection::registerServiceWorkerClientInternal):
(WebKit::WebSWServerConnection::unregisterServiceWorkerClient):
(WebKit::WebSWServerConnection::computeThrottleState const):
(WebKit::WebSWServerConnection::updateThrottleState):
(WebKit::WebSWServerConnection::contextConnectionCreated):
(WebKit::WebSWServerConnection::sendToContextProcess):
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.h:
(isType):
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.h:
(isType):
* Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServerToContextConnection.cpp:
(WebKit::WebSharedWorkerServerToContextConnection::WebSharedWorkerServerToContextConnection):
(WebKit::WebSharedWorkerServerToContextConnection::launchSharedWorker):
* Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.cpp:
(WebKit::NetworkRTCProvider::NetworkRTCProvider):
(WebKit::NetworkRTCProvider::attributedBundleIdentifierFromPageIdentifier):
* Source/WebKit/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebKit/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebKit/SaferCPPExpectations/UncountedLambdaCapturesCheckerExpectations:
* Source/WebKit/Shared/API/Cocoa/_WKRemoteObjectRegistry.mm:
(-[_WKRemoteObjectRegistry _callReplyWithID:blockInvocation:]):
* Source/WebKit/Shared/ApplePay/WebPaymentCoordinatorProxy.cpp:
(WebKit::WebPaymentCoordinatorProxy::showPaymentUI):
* Source/WebKit/UIProcess/AuxiliaryProcessProxy.cpp:
(WebKit::AuxiliaryProcessProxy::shutDownProcess):
* Source/WebKit/WebProcess/Storage/WebServiceWorkerFetchTaskClient.cpp:
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveRedirection):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveResponse):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveDataInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveFormDataAndFinishInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didReceiveBlobChunk):
(WebKit::WebServiceWorkerFetchTaskClient::didFailInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didFinishInternal):
(WebKit::WebServiceWorkerFetchTaskClient::didNotHandleInternal):
(WebKit::WebServiceWorkerFetchTaskClient::usePreload):
(WebKit::WebServiceWorkerFetchTaskClient::contextIsStopping):
* Source/WebKit/WebProcess/Storage/WebServiceWorkerFetchTaskClient.h:

Canonical link: https://commits.webkit.org/288561@main
